biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/configuration/BndConfiguration.java

```diff
@@ -111,6 +280,13 @@ private File loadProjectProperties(Processor processor, MavenProject bndProject,
 		return pomFile;
 	}
 
+	private Optional<Xpp3Dom> getConfiguration(MavenProject mavenProject) {
+		if (mavenProject == null) {
+			return Optional.empty();
+		}
+		return getConfiguration(mavenProject.getBuildPlugins()).or(() -> getConfiguration(mavenProject.getParent()));
```

Comment:
I think this here needs to also look in the parentage for properties. The failure stems from not finding the output time stamp in the parent pom of the project. So it is configuration(parentProject)+configuration(thisProject). Also need to allow the descendant projects to override the ancestor projects.

Reply:
Yes I think you are right here, I was hoping this is already resolved by maven but it seems not be the case...